src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
```

Comment:
> ```javascript
> const isSessionOpen = $(textField).data('isSessionOpen') || true;
> ```

@whipermr5 we can't use `true` as the second value because in pipe operator

- If the first value is `false`, it checks the second value. If it's `true`, it returns `true` and if it's `false`, it returns `false`.

- If the first value is `true`, it always returns `true`, no matter what the second value is.

***

Let us consider our case in which if the `$(textField).data('isSessionOpen')` value is `false` that means the session is closed. But still the `const isSessionOpen` value will evaluate to `true` because of this :point_down:

```javascript
const isSessionOpen = $(textField).data('isSessionOpen') || true;
```
_Set `$(textField).data('isSessionOpen')` value to `false`_

```javascript
const isSessionOpen = false || true;
```
Now that we have `false` as our first value and `true` as our second value, the `||` operator will evaluate to `true` which means that the `const isSessionOpen` value will be `true` which should have been `false`.

Not very good in explaining, but I hope you got this :sweat_smile: